trailing_spaces.py

```diff
@@ -96,7 +94,10 @@ def find_trailing_spaces(view):
 
     offending_lines = view.find_all(regexp if include_empty_lines else no_empty_lines_regexp)
 
-    if include_current_line:
+    sel = view.sel()
+    line = len(sel) and view.line(sel[0].b)
```

Comment:
I did not know you could do this with `int`s.

Reply:
`view.sel()` returns `Selection` object. Do you mean that you could do `len()` on it?

Reply:
I mean `and`s to return the last one.

Reply:
Right, documented here https://docs.python.org/3/reference/expressions.html#boolean-operations